Components/Image/image.css

```diff
@@ -0,0 +1,14 @@
+.img {
+  height: 9.375rem;
+  width: 9.375rem;
+}
+
+.round {
+  border-radius: 50%;
+}
```

Comment:
My suggestion would be to put this class into utility stylesheet, so as to keep up with conventions.

Reply:
I have updated my code please review it again